synapse/federation/federation_client.py

```diff
@@ -536,8 +541,9 @@ def make_membership_event(self, destinations, room_id, user_id, membership,
             params (dict[str, str|Iterable[str]]): Query parameters to include in the
                 request.
         Return:
-            Deferred: resolves to a tuple of (origin (str), event (object))
-            where origin is the remote homeserver which generated the event.
+            Deferred[tuple[str, dict, int]]: resolves to a tuple of
+            `(origin, event, event_format)` where origin is the remote
+            homeserver which generated the event.
```

Comment:
could you clarify that `event_format` is one of the values in `synapse.api.constants.EventFormatVersions` ?